rowan/components/scripts/postLoad.st

```diff
@@ -0,0 +1,11 @@
+| symbolDictionary |
+
+symbolDictionary := Rowan image 
+	symbolDictNamed: 'Kepler'
+	ifAbsent: [Error signal: 'Kepler symbol dictionary missing' ].
+
+"Global variables definition"
+symbolDictionary at: #SystemInterfaces put: Namespace new.
+
+"Register Kepler's built-in system interfaces"
+SubsystemImplementation allSubclassesDo: [:each | each registerInterfaces].
```

Comment:
I think it's better here to be explicit, because when Kepler is a dependency of another project this code is executed after all the code is loaded and it can include other implementations of subsystems not necessarily registering the interfaces in this way.